### ISSUE TITLE
Ammonia/ammonium production rates

### ISSUE THREAD

**Issue by gwemon** [nvs-vocabs/ereefs#33]
Ammonia production (model: 'NH4_pr', sensor: '', standard: ''')
Ammonia production in Sediment, Ammonia production sediment (model: 'NH4_pr_sed', sensor: '', standard: ''')

I looked for these in the paper: https://gmd.copernicus.org/articles/13/4503/2020/ but could not find any references to "ammonia". Presumably this parameter should be read as "ammonium production"?

What are the processes covered by these parameters? any ammonia/um producing processes or any one in particular? excretion presumably but also decomposition of organic nitrogen (i.e. ammonification)?

@sharon-tickell

**Comment by markebaird** [comment]
Yes, it should be ammonium production. And yes, all those processes add to ammonium production in the sediment.
